ManagementAPI/Private/CMAClient.m

```diff
@@ -140,13 +141,11 @@ -(id)initWithAccessToken:(NSString *)accessToken configuration:(CDAConfiguration
             configuration.server = @"api.contentful.com";
         }
 
-        if (!configuration.userAgent) {
-            configuration.userAgent = @"contentful-management.objc/0.10.0";
-        }
-
         self.client = [[CDAClient alloc] initWithSpaceKey:nil
                                               accessToken:accessToken
-                                            configuration:configuration];
+                                            configuration:configuration
+                                             isCMARequest:YES];
```

Comment:
oh yeah, the other thing is that the only RequestSerializer is technically a class in the CDA